applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/app_ZDebug.conf

```diff
@@ -96,6 +99,16 @@ CONFIG_ADXL362=y
 CONFIG_ADXL362_ACCEL_RANGE_8G=y
 CONFIG_ADXL362_ACCEL_ODR_100=y
 
+CONFIG_MCUMGR=y
```

Comment:
why not under `Bootloader Configuration` section?

Reply:
I don't think it's worth a CI rerun but we may want to move it on next occasion.

Reply:
Ok I could keep mcumgr related configuration in `Bootloader Configuration` section.

Reply:
Maybe we could even introduce a Kconfig option that selects most of these or add a common application overlay. That could simplify app/sample configuration a lot and it should not be difficult to add.

Reply:
maybe but we can do it as a separate thing. I also don't know where it would land